packages/webapp/src/components/Form/NumberInput/useNumberInput.ts

```diff
@@ -55,6 +55,10 @@ export type NumberInputOptions = {
    * - If input value equals min then decrementing with stepper and keyboard is disabled.
    */
   min?: number;
+  /**
+   * Controls whether or not to clamp value on blur that is outside of allowed range
+   */
+  clampOnBlur?: boolean;
```

Comment:
Added this so that the error remains after the input is blurred as it is shown in the designs